pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -205,11 +197,25 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
           component: (
             <CustomRules
               customRulesFiles={customRulesFiles}
-              setValue={(val) => setValue("customRulesFiles", val)}
+              sources={sources}
+              setCustomRulesFiles={(val) => setValue("customRulesFiles", val)}
```

Comment:
Can we change these function names & values to something more descriptive than setValue, val ?